Production Compilation Question

First, I'm totally new to react and webpack. Thank you for this excellent, detailed example of usage for me to learn from.
Question:
When I `npm run build` I get the following output:

```
    ERROR in ./config/prerender.jsx?Main
    Module not found: Error: Cannot resolve module 'html-loader' in <abspath>/react-test/config
     @ ./config/prerender.jsx?Main 5:11-43
```

Does this have to do with ["loaders resolved relative to the resource their applied to..."](http://webpack.github.io/docs/configuration.html#module-loaders)?

[Full error output with --display-error-details](https://gist.github.com/localnerve/12ae32a69a9cbae753ff)
Thanks in advance,
Alex

--- a/package.json
+++ b/package.json
@@ -34,6 +34,7 @@
     "stylus-loader": "^0.4.0",
     "less": "^1.7.4",
     "less-loader": "^0.7.5",
-    "null-loader": "^0.1.0"
+    "null-loader": "^0.1.0",
+    "html-loader": "^0.2.2"
   }
 }

